### PR TITLE
fix: fix hover style

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | fix: fix hover style |
| 🇨🇳 Chinese | 修复分页器简洁模式中的上一页下一页按钮 hover 样式丢失的问题 |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 547e9d7</samp>

Improve pagination component's usability and appearance by adding hover and active states for buttons in `components/pagination/style/index.tsx`.

### <samp>🤖 Generated by Copilot at 547e9d7</samp>

*  Add hover and active states for disabled items and buttons in simple pagination style ([link](https://github.com/ant-design/ant-design/pull/41685/files?diff=unified&w=0#diff-ddac945eb46552285af334850cebb6d4ab21a6d9a6b603b3869334c9cc228c66L101-R108), [link](https://github.com/ant-design/ant-design/pull/41685/files?diff=unified&w=0#diff-ddac945eb46552285af334850cebb6d4ab21a6d9a6b603b3869334c9cc228c66R124-R133))
*  Add hover and active state for jump-prev and jump-next buttons in default pagination style ([link](https://github.com/ant-design/ant-design/pull/41685/files?diff=unified&w=0#diff-ddac945eb46552285af334850cebb6d4ab21a6d9a6b603b3869334c9cc228c66L227-R243))

### CODE REVIEW

File: components/pagination/style/index.tsx [hunk 1]
```diff
@@ -120,6 +121,16 @@ const genPaginationDisabledStyle: GenerateStyle<PaginationToken, CSSObject> = (t
         color: token.colorTextDisabled,
       },
     },
+    [`
+      &${componentCls}-simple ${componentCls}-prev,
+      &${componentCls}-simple ${componentCls}-next
+    `]: {
+      [`&${componentCls}-disabled ${componentCls}-item-link`]: {
+        '&:hover, &:active': {
+          backgroundColor: 'transparent',
```
Comment (li-jia-nan) [inline components/pagination/style/index.tsx:130]: 上面权重不够，所以这里再写一遍

Comment (afc163) [reply components/pagination/style/index.tsx:130]: 感觉代码重复比较多，看看有没有更简洁的优先级修改方式。

Comment (li-jia-nan) [reply components/pagination/style/index.tsx]: 稍微简化了一下

File: components/pagination/style/index.tsx [hunk 2]
```diff
@@ -120,6 +121,15 @@ const genPaginationDisabledStyle: GenerateStyle<PaginationToken, CSSObject> = (t
         color: token.colorTextDisabled,
       },
     },
+    [`&${componentCls}-simple`]: {
+      [['prev', 'next'].map((cls) => `${componentCls}-${cls}`).join(',')]: {
```
Comment (MadCcc) [inline components/pagination/style/index.tsx:125]: 建议直接写。。这样不太直观

Comment (li-jia-nan) [reply components/pagination/style/index.tsx]: done~

**Review by afc163 — APPROVED** [review]

**Pull request merged**